tests/ui/perfherder/alerts_test.jsx

```diff
@@ -510,8 +510,7 @@ test('Correct message is displayed if the framework id is invalid', async () =>
 test('Selecting `all` from framework button does not filter by framework', async () => {
```

Comment:
This old test's description isn't correct enough, because we're testing the `all *` from both status & framework dropdowns. It'd be better to address it here.

Reply:
@ionutgoldan I noticed that as well--will update it

Reply:
when updating the test description, should the tests for `frameworks` and `status` be separate or is it fine to keep them together?

also--is the description what is displayed when a test *fails* or _passes_?